### ISSUE TITLE
Incorrect Format Table

### ISSUE THREAD

**Issue by lrAlval** [simov/express-admin#88]
![cli](https://cloud.githubusercontent.com/assets/13581033/9614263/56387838-50b6-11e5-81f5-85f3004dbf5a.jpg)

is ther a way to change this format ? with out altering the whole format of others tables ? 

thanks in
advance

**Comment by simov** [comment]
Yes, you can override these [styles](https://github.com/simov/express-admin/blob/master/public/express-admin.css#L18-L22).
